### DISCUSSION TITLE
Gold Rush Event!

### DISCUSSION THREAD

**Discussion by ulyssesjmantes** [sunflower-land/sunflower-land#1554] [Ideas]
This is to propose an event before the land expansion happens. This event will be of a very limited time, 3 days maybe, where humans discovered an abundant silver and gold mines (rocks maybe included) before being defeated and expelled by goblins from sunflower land. As a result, resources mining cooldown is greatly reduced. Example, 30mins for silver, 1hr for gold, giving XP boost on tools and much needed resources for land expansion. 

Another way is by using the Travelling Salesman where the salesman will accept a certain number of tools or mined resources (rock, silver, gold) in exchange of a forbidden knowledge, some kind of "XP Book", that will give an additional XP for Tools.

This event will be a win-win both for players and developers. For players who have been mining since day 1 to get the level 10 on Tools skills (Gold Rush particularly) will have some boost, and for those who have stocks of crafting tools will be able to use them before land expansion happens. New players will have some chance also to maybe get at least the level 5 skill of Tools. For developers, this event will greatly increase the transactions, pumping both MATIC revenue generation and SFL value. 

Thank you!

**Comment by SuperSynapse** [comment]
I'm here with you, currently just below 4,500. Been active since SFL launched working toward Gold Rush as a personal goal.

Would be heartbroken not to make it, but giving it my all. I'm a slave to my phone timer.

**Comment by future2back** [comment]
I talked about the need for an incentive in #1474 as these were never affected by the halvening.

**Comment by adamhannigan** [comment]
@ulyssesjmantes This is a great idea and the team love it!

Unfortunately we are not adding new functionality/gameplay until we release Land Expansion. Every new feature we add, means that it will take longer to launch Land Expansion 😅 

I hope you understand 🙏

**Reply by future2back** [reply]
@adamhannigan exp event is enought to burn resources and token.
